builder/comp-builder.nix

```diff
@@ -60,7 +60,7 @@ let self =
 
 # Profiling
 , enableLibraryProfiling ? component.enableLibraryProfiling
```

Comment:
Should we also ditch this option and just have one profiling option?

Reply:
It depends whether or not it makes sense to only enable profiling in the library and not in the executables. I personally would keep both options as they are both present in cabal, and by name `enableProfiling` already looks more general.

The generated libraries (in the three flavors `.a` `_p.a` and `.so`) are identical when using `--enable-profiling` and `--enable-library-profiling` and `--enable-profiling --disable-library-profiling`, and the executable differs, so it seems that `--(enable|disable)-profiling` overrides `--(enable|disable)-library-profiling`.